libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.h

```diff
@@ -399,6 +426,11 @@ extern NSString * const kSFAuthenticationManagerFinishedNotification;
  */
 - (void)dismissAuthViewControllerIfPresent;
 
+/** Return SFOAuthCredentials using know oauthClientId,
+    @return Connect App auth credentials
+ */
+-(SFOAuthCredentials *)oauthCredentials;
```

Comment:
Minor: can we rename this a bit, e.g. `createOAuthCredentials`?  The current name makes it feel like it's returning something that already exists.